src/main/java/mate/academy/springboot/datajpa/service/ProductService.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.springboot.datajpa.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.datajpa.model.Product;
+
+public interface ProductService {
+    Product create(Product product);
+
+    Product getById(Long id);
+
+    Product update(Product product);
+
+    void deleteById(Long id);
+
+    List<Product> getProductsByPriceBetween(BigDecimal fromPrice, BigDecimal toPrice);
```

Comment:
```suggestion
    List<Product> getProductsByPriceBetween(BigDecimal from, BigDecimal to);
```

Reply:
Same

Reply:
I think you don't need write more informative than (from, to) cause method name means price here